framework/Samplers/LimitSurfaceSearch.py

```diff
@@ -43,6 +44,16 @@
 from utils import InputData
 #Internal Modules End--------------------------------------------------------------------------------
 
+def _toStr(s):
+  """
+    Removes unicode from strings in Python 2 so amsc can use it.
+    @ In, s, unicode or str, String to convert to plain str
+    @ Out, s, str, Converted str
+  """
+  if sys.version_info.major > 2:
+    return s
+  return s.encode('ascii','ignore')
```

Comment:
as above. Place this in the utils.py so we are not redundant

Reply:
Removed.